### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by devBayo**

### CODE REVIEW

**Comment by manjunani** [comment]
There are some other typo mistakes as well please do resolve them

**Comment by manjunani** [comment]
and also i did mentioned that the images of the contributors are not coming correctly please do work on them

**Comment by devBayo** [comment]
How about now?
Kindly check again

**Comment by devBayo** [comment]
Kindly add the hacktoberfest tag.

**Comment by manjunani** [comment]
It Looks Great man  and by the way can u work on this one too https://contrib.rocks/image?repo=Xtreme-Thinkers/hacktoberfest2022 i did mentioned that in the issue and by the i can see isue is being there in the readme file instead of issue can u correct that as well so that i can merge the PR

**Comment by devBayo** [comment]
I'm not sure I understand what the issue is.
Can you explain better?
Can you create it as a new issue then assign it to me.

Also you haven't merged my pull request.

File: README.md [hunk 1]
```diff
@@ -40,7 +39,7 @@
 [Rebasing and Updating Pull Request](https://www.digitalocean.com/community/tutorials/how-to-rebase-and-update-a-pull-request)
 
 ### Star and Fork this Repository 👇🏻
-###### ⚡  Do Raise an isue by yourself to this repo of (what changes that you wanted to do / what new features or projects that you want to implement) and then later based on my review i will assign the issue to you then later you can probably make your changes and raise a PR.
+###### ⚡  Raise an isue by yourself to this repo of (what changes can be made / what new features can be implemented) and then later based on my review i will assign the issue to you then later you can probably make your changes and raise a PR.
```
Comment (manjunani) [inline README.md:42]: isue should be corrected to issue

**Review by manjunani — COMMENTED** [review]
Isue  in line 42 should be corrected to issue

**Comment by manjunani** [comment]
> I'm not sure I understand what the issue is. Can you explain better? Can you create it as a new issue then assign it to me?
> 
> Also you haven't merged my pull request.

as there were some typo mistakes i had to keep the PR Open until you make the changes bro..

**Comment by devBayo** [comment]
I've made changes, kindly check it out.

**Comment by manjunani** [comment]
Thanks Buddy, will merge the changes now and by the way the contributor issue is working fine so i wont be raising the issue as of now

**Review by manjunani — APPROVED** [review]

**Comment by devBayo** [comment]
Thanks buddy 😊

**Pull request merged**